internal/core/context.rs

```diff
@@ -20,7 +20,7 @@ pub(crate) struct SlintContextInner {
     pub(crate) window_count: core::cell::RefCell<isize>,
     /// This property is read by all translations, and marked dirty when the language change
     /// so that every translated string gets re-translated
```

Comment:
```suggestion
    /// This property is read by all translations, and marked dirty when the language changes,
    /// so that every translated string gets re-translated. The property's value is the current selected
    /// language when bundling translations.
```